src/app/src/main/kotlin/org/imperial/mrc/hint/Annotations.kt

```diff
@@ -0,0 +1,6 @@
+package org.imperial.mrc.hint
+
+@Retention(AnnotationRetention.BINARY)
+annotation class ExcludeFromGeneratedCodeCoverage
+
+typealias NoCoverage = ExcludeFromGeneratedCodeCoverage
```

Comment:
Data classes are tested in the parent class i.e LogMetadata. Children data classes are not used directly and are just for serialisation. In fact these children classes are tested in GenericLoggerImplTests. Code coverage was complaining because they had been tested within LogMetadata.

Reply:
I just noticed we do have this in OW, so I guess there's precedent (but I probably complained about that too! 😆) - I think that is a slightly different use case as that's all for types passed through from Orderly Server rather than objects created and used internally.

Reply:
I have now removed noCoverage annotations.